### DISCUSSION TITLE
Cps counter as BPM

### DISCUSSION THREAD

**Discussion by Anymusxd** [ppy/osu#30576] [Ideas]
First of all, I know there is a bpm counter in-game already, but that just shows the map bpm, not your current fingers/tapping bpm

My idea is for the cps counter to have an alternative option for real time bpm counter

Again, it's for the player's tapping, not the beatmap's bpm

Unless there's an option I'm not aware of, that can always be a case

**Comment by Joehuu** [comment]
This was rejected in https://github.com/ppy/osu/pull/28852 because the unit naming didn't make sense.
